app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/savedReplySelector.tsx

```
@@ -0,0 +1,112 @@
+          <span className="flex items-center gap-2">
+            <SavedReplyIcon className="h-4 w-4" />
+            <span>Insert saved reply</span>
+          </span>
+          <ChevronDown className="ml-2 h-4 w-4 shrink-0 opacity-50" />
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent
+        className="w-full p-0"
+        align="start"
+        style={{ width: buttonRef.current?.offsetWidth }}
+      >
```

ESLint

[error] Replace `⏎········className="w-full·p-0"⏎········align="start"⏎········style={{·width:·buttonRef.current?.offsetWidth·}}⏎······` with `·className="w-full·p-0"·align="start"·style={{·width:·buttonRef.current?.offsetWidth·}}`

(prettier/prettier)